### ISSUE TITLE
Stored Object: paper:openreview.esYrEndGsr

### ISSUE THREAD

**Issue by yanaiela** [yanaiela/papers-feed#1018]
{
  "sourceId": "openreview",
  "paperId": "esYrEndGsr",
  "url": "https://openreview.net/forum?id=esYrEndGsr",
  "title": "Influence Functions for Scalable Data Attribution in Diffusion Models",
  "authors": "Bruno Kacper Mlodozeniec, Runa Eschenhagen, Juhan Bae, Alexander Immer, David Krueger, Richard E. Turner",
  "abstract": "Diffusion models have led to significant advancements in generative modelling. Yet their widespread adoption poses challenges regarding data attribution and interpretability. In this paper, we aim to help address such challenges in diffusion models by extending influence functions. Influence function-based data attribution methods approximate how a model's output would have changed if some training data were removed. In supervised learning, this is usually used for predicting how the loss on a particular example would change. For diffusion models, we focus on predicting the change in the probability of generating a particular example via several proxy measurements. We show how to formulate influence functions for such quantities and how previously proposed methods can be interpreted as particular design choices in our framework. To ensure scalability of the Hessian computations in influence functions, we use a K-FAC approximation based on generalised Gauss-Newton matrices specifically tailored to diffusion models. We show that our recommended method outperforms previously proposed data attribution methods on common data attribution evaluations, such as the Linear Data-modelling Score (LDS) or retraining without top influences, without the need for method-specific hyperparameter tuning.",
  "timestamp": "2025-12-29T09:54:26.831Z",
  "rating": "novote",
  "publishedDate": "22 Jan 2025",
  "tags": [
    "diffusion models",
    "influence functions",
    "Generalised Gauss Newton",
    "GGN",
    "data attribution",
    "Hessian approximation",
    "interpretability",
    "curvature",
    "Kronecker-Factored Approximate Curvature",
    "K-FAC"
  ],
  "doi": "",
  "journalName": "ICLR 2025 Oral",
  "sourceType": "url"
}

**Comment by yanaiela** [comment]
{
  "_data": {
    "sourceId": "openreview",
    "paperId": "esYrEndGsr",
    "url": "https://openreview.net/forum?id=esYrEndGsr",
    "title": "Influence Functions for Scalable Data Attribution in Diffusion Models",
    "authors": "Bruno Kacper Mlodozeniec, Runa Eschenhagen, Juhan Bae, Alexander Immer, David Krueger, Richard E. Turner",
    "abstract": "Diffusion models have led to significant advancements in generative modelling. Yet their widespread adoption poses challenges regarding data attribution and interpretability. In this paper, we aim to help address such challenges in diffusion models by extending influence functions. Influence function-based data attribution methods approximate how a model's output would have changed if some training data were removed. In supervised learning, this is usually used for predicting how the loss on a particular example would change. For diffusion models, we focus on predicting the change in the probability of generating a particular example via several proxy measurements. We show how to formulate influence functions for such quantities and how previously proposed methods can be interpreted as particular design choices in our framework. To ensure scalability of the Hessian computations in influence functions, we use a K-FAC approximation based on generalised Gauss-Newton matrices specifically tailored to diffusion models. We show that our recommended method outperforms previously proposed data attribution methods on common data attribution evaluations, such as the Linear Data-modelling Score (LDS) or retraining without top influences, without the need for method-specific hyperparameter tuning.",
    "timestamp": "2025-12-29T09:54:26.831Z",
    "rating": "novote",
    "publishedDate": "22 Jan 2025",
    "tags": [
      "diffusion models",
      "influence functions",
      "Generalised Gauss Newton",
      "GGN",
      "data attribution",
      "Hessian approximation",
      "interpretability",
      "curvature",
      "Kronecker-Factored Approximate Curvature",
      "K-FAC"
    ],
    "doi": "",
    "journalName": "ICLR 2025 Oral",
    "sourceType": "url"
  },
  "_meta": {
    "client_version": "0.11.1",
    "timestamp": "2025-12-29T09:54:27.346Z",
    "update_mode": "append",
    "issue_number": 1018
  },
  "type": "initial_state"
}
